### ISSUE TITLE
As an LMA User I want a relationship from the Lead back to the License so that I can use XOF fields to run marketing automation from the Lead

### ISSUE THREAD

**Issue by jengler74** [CodeScience/BOSS#7]
from Arrowpointe

**Comment by ronkiker** [comment]
@jengler74 - i see the mention of XOF throughout many of these tickets. Are the XOF mentioned above those fields?  Does we have a whole list of all the XOF fields and their Objects?

**Comment by jengler74** [comment]
Correct, the ones listed in the details are the ones to pull from the License

**Comment by ronkiker** [comment]
- L/U first
- XOF second

**Comment by ronkiker** [comment]
License Status is a pick-list

**Comment by ronkiker** [comment]
@jengler74  - Two fields are NOT present within the LMA version we have: Sandbox/Production AND Salesforce Edition.  Can you get an updated version of the LMA for Boss PKG?
